tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -30,7 +30,7 @@ images are listed in the corresponding installation sections.
 If you encounter installation errors, see
 [common problems](#common_install_problems) for some solutions.
```

Comment:
Done.  Note that this only fixes os_setup.md, and any inward reference files (so that there are no broken links).  If I fix all files I touched it will quickly expand to all files :-)